main.go

```diff
@@ -35,7 +35,7 @@ const (
 var (
 	// Flags:
 	tomcatFlag = flag.Bool("tomcat", false, `Choose to build a Bonita Tomcat bundle containing your application
-use -bonita-tomcat-bundle to specify the path (on your machine) to the Bonita tomcat bundle file (Bonita*.zip); otherwise the file is looked for in the current folder`)
```

Comment:
Why removing this?

Reply:
I think it is redundant and we don't precise this on the other options.